pythia/common/trainer.py

```diff
@@ -128,12 +128,13 @@ def load_model(self):
 
         if "cuda" in str(self.config.training_parameters.device):
             rank = self.local_rank if self.local_rank is not None else 0
-            self.writer.write(
-                "CUDA Device {} is: {}".format(
-                    rank, torch.cuda.get_device_name(self.local_rank)
-                )
+            device_info = "CUDA Device {} is: {}".format(
+                rank, torch.cuda.get_device_name(self.local_rank)
             )
 
+            self.writer.write(device_info)
+            print(device_info)
```

Comment:
NIT: If possible use logging here instead of print statements.

Reply:
Will be fixed, requires some changes in logger.